Solutions/Marain.Tenancy.Cli/Marain/Tenancy/Cli/Commands/Create.cs

```diff
@@ -73,7 +73,7 @@ public async Task OnExecute(CommandLineApplication app)
 
             if (string.IsNullOrEmpty(this.Name))
             {
-                throw new ArgumentException("Name must be supplied", nameof(this.Name));
+                throw new InvalidOperationException("Name must be supplied");
```

Comment:
Not specifically net6 but I'm really glad you fixed this.

Reply:
It was something the new analysers in the latest SDK picked up, which was my reasoning for this getting in there as a .net 6 change.